awslimitchecker/checker.py

```diff
@@ -218,6 +218,19 @@ def get_project_url(self):
         """
         return self.vinfo.url
 
+    def remove_skipped_services(self, service_to_skip=None):
+        """
+        If ``service_to_skip`` is specified, the listed services
+        are removed from the services[] class list.
+
+        :param service_to_skip: the name(s) of one or more services to avoid
+          returning limits for
+        :type service_to_skip: list
+        """
+        if service_to_skip is not None:
```

Comment:
function signature updated as suggested.
`if` removed.
`service_to_skip` variable name changed to `services_to_remove`.